lib/processors/bundlers/moduleBundler.js

```diff
@@ -75,6 +75,10 @@ const log = getLogger("builder:processors:bundlers:moduleBundler");
  * @property {boolean} [declareRawModules=false] Whether raw modules should be declared after jQuery.sap.global
  *		became available. With the usage of the ui5loader, this flag should be set to 'false'
  * @property {boolean} [sort=true] Whether the modules should be sorted by their dependencies
+ * @property {boolean} [async=true] Whether the `require` section of the module should be loaded asynchronously.
+ * 		When set to true, the modules are loaded using `sap.ui.require` instead of `sap.ui.requireSync`.
```

Comment:
Should we mention that each section creates one sap.ui.require call (or alternatively, that the modules are required in parallel)?